library/alloc/src/boxed.rs

```diff
@@ -1501,21 +1499,48 @@ impl<A: Allocator> Box<dyn Any, A> {
     /// print_if_string(Box::new(my_string));
     /// print_if_string(Box::new(0i8));
     /// ```
+    #[inline]
+    #[stable(feature = "box_send_sync_any_downcast", since = "1.51.0")]
```

Comment:
Looks like this one was an over-aggressive copy-paste.
```suggestion
    #[stable(feature = "rust1", since = "1.0.0")]
```
<https://doc.rust-lang.org/1.0.0/std/boxed/struct.Box.html#method.downcast>)